CHANGELOG.md

```diff
@@ -140,7 +140,7 @@
   - More consistent handling of missing keys: if a config key is missing pureconfig always throws a
     `KeyNotFoundException` now, unless the `ConfigConvert` extends the new `AllowMissingKey` trait.
   - Add support for the `java.time` package. Converters types which support different string formats, such as `LocalDate`,
-    must be configured before they can be used. See the [README](https://github.com/melrief/pureconfig#configurable-converters)
+    must be configured before they can be used. See the [README](https://pureconfig.github.io/configurable-converters.html)
```

Comment:
Shouldn't you replace README with documentation here?

Reply:
fixed with 915c002